scripts/add_cypress_test_users.ts

```diff
@@ -243,6 +245,12 @@ async function main() {
             email: 'shi-tong@example.com',
             role: 'BUSINESSOWNER_USER' as const,
             state: undefined,
+        },
+        {
+            name: 'Azula',
+            email: 'azula@example.com',
```

Comment:
please match the familyName given elsewhere. Need users setup between cypress and local to match